src/js/modules/articles/actions.js

```diff
@@ -1,11 +1,67 @@
 import axios from 'axios';
 import * as t from './actionTypes';
+import { getSections } from '../sections/selectors';
 

```

Comment:
Import the `sections` module up here, and then when you use the selector, do `sections.selectors.getSections`. That way, others know where the selector is coming from.